src/libmongoc/tests/json/server_discovery_and_monitoring/unified/serverMonitoringMode.json

```diff
@@ -0,0 +1,459 @@
+{
+  "description": "serverMonitoringMode",
+  "schemaVersion": "1.17",
+  "runOnRequirements": [
+    {
+      "topologies": [
+        "single",
+        "sharded",
+        "sharded-replicaset"
+      ],
+      "serverless": "forbid"
+    }
+  ],
+  "tests": [
+    {
+      "description": "connect with serverMonitoringMode=auto >=4.4",
+      "runOnRequirements": [
+        {
+          "minServerVersion": "4.4.0"
+        }
+      ],
+      "operations": [
+        {
+          "name": "createEntities",
+          "object": "testRunner",
+          "ignoreResultAndError": true,
```

Comment:
`"ignoreResultAndError": true` was added across this spec test to prevent the test runner from checking the result of an operation that doesn't have one (all instances of `createEntities` and `waitForEvent`). Open to suggestions on how to fix this issue without modifying the spec test. Maybe we can add a conditional in `operation.c` where `ignore_result_and_error` is checked but this feels a little clunky.